Library/Homebrew/cmd/cleanup.rb

```diff
@@ -62,6 +62,10 @@ def cleanup_cache
         cleanup_path(path) { path.unlink }
         next
       end
+      if path.to_s.include?("java_cache")
```

Comment:
This style was mimicked from below (I have a thing for consistency, heh). I've no strong objection to checking the path is actually a path, but I think the chances of stumbling across a file including `java_cache` that resides inside our `HOMEBREW_CACHE` is pretty low.